src/dlsnode/util/aio/AsyncIO.d

```diff
@@ -204,6 +205,74 @@ class AsyncIO
         }
     }
 
+    /***************************************************************************
+
+        Set of non-blocking methods. The user is responsible to suspend the fiber,
+        and AsyncIO will not resume it. Instead, the callback will be called
+        where the user can do whatever is required.
+
+        TODO: this is a class, since there's no outer.
```

Comment:
This doesn't make sense.

Reply:
I should make this a little bit clearer. This shouldn't really be a class, it should be a struct, but I couldn't make it a struct, as it don't have `outer` pointer (if I remember correctly). Now you put my attention back to it, it can be `final class`.